README.md

```diff
@@ -33,7 +33,7 @@ This repository contains the Ansible scripts for installing and configuring WSO2
 │   │   ├── amazon-corretto-8.202.08.2-linux-x64.tar.gz
```

Comment:
Change the Coretto version to `222.10.1` in here as well

Reply:
Fixed in https://github.com/wso2/ansible-is/pull/35/commits/50b37908cae9d75f4ebb88586c88f07efedc2f03.